lib/elixir/lib/gen_server.ex

```diff
@@ -159,6 +159,71 @@ defmodule GenServer do
   the GenServer callbacks as doing so will cause the GenServer to misbehave.
   If you want to receive custom messages, always receive them in `handle_info/2`.
 
+  ## Debugging with the :sys module
+
+  GenServers, as all [special processes](http://erlang.org/doc/design_principles/spec_proc.html)
```

Comment:
"GenServers, as [special processes](...)," would be better phrasing, IMO, but in any event a comma is definitely missing. :)